root.tf

```diff
@@ -673,6 +673,32 @@ module "bastion_role" {
   policy_attachments = {}
 }
 
+module "update_waf_and_security_groups_policy" {
```

Comment:
I'd be tempted to move the specific github modules into a separate root file by themselves: `root_github` ?

Reply:
Yeah that makes sense, the main root file is getting a bit big.